scripts/workflow.bm.R

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env Rscript
```

Comment:
This script and a couple others are just part of the development process. Yes, the plan is to fully integrate this into the overall workflow as things progress, but it's not at that stage of maturity yet.